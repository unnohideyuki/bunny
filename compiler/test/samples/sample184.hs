x :: Double
x = 123456789012345678901234567890 :: Double

y :: Double
y = 3

main = do print (x + y)
          print (x - y)
          print (x * y)
          print (signum (1 :: Double))
          print (signum (0 :: Double))
          print (signum (-1 :: Double))

