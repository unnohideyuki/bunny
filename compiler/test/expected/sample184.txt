1.2345678901234568e29
1.2345678901234568e29
3.70370367037037e29
1.0
0.0
-1.0
